### PR TITLE
Additional Sink tests

### PR DESCRIPTION
**Pull request by poojantcs**

### CODE REVIEW

File: src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java [hunk 1]
```diff
@@ -45,18 +45,27 @@ public class TestSetupHooks {
   public static String bqTargetTable = StringUtils.EMPTY;
   public static String bqSourceDataset = "ServiceNowTestDS";
   public static String bqSourceTable;
+  public static String systemId;
+  public static String receivingSlipLineRecordUniqueNumber;
 
+//    ServiceNowPropertiesPageActions.fillCredentialsSectionProperties(
```
Comment (poojantcs) [inline src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java]: Remove the commented lines

File: src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java [hunk 2]
```diff
@@ -69,16 +78,180 @@ public static void createRecordInProductCatalogItemTable() throws UnsupportedEnc
...
+    BigQueryClient.getSoleQueryResult("create table `" + bqSourceDataset + "." + bqSourceTable + "` as " +
+                                        "SELECT * FROM UNNEST([ STRUCT((DATE '2022-06-10') AS purchase_date," +
+                                        " (DATETIME '2022-06-08 16:00:00') AS end_date," +
+                                        " (TIMESTAMP '2022-05-10 15:00:00-08:00') AS start_date," +
+                                        " ' " + systemId + " ' AS sys_id)])");
+
+    BeforeActions.scenario.write("BQ source Table " + bqSourceTable + " created successfully");
+  }
+
+
```
Comment (poojantcs) [inline src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java]: Remove the blank lines

File: src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIClient.java [hunk 3]
```diff
@@ -82,6 +83,29 @@ protected RestAPIResponse executePost(RestAPIRequest request) throws Unsupported
     return apiResponse;
   }
 
+  /**
+   * Executes the Rest API request and returns the response.
+   *
+   * @param request the Rest API request
+   * @return an instance of RestAPIResponse object.
+   */
+  protected RestAPIResponse executePatch(RestAPIRequest request) throws UnsupportedEncodingException {
```
Comment (poojantcs) [inline src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIClient.java]: Are we using this in any test?

File: src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java (hunk 1 above)
Comment (Vipinofficial11) [reply src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java]: Done

File: src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java (hunk 2 above)
Comment (Vipinofficial11) [reply src/e2e-test/java/io/cdap/plugin/tests/hooks/TestSetupHooks.java]: Done

File: src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIClient.java (hunk 3 above)
Comment (Vipinofficial11) [reply src/main/java/io/cdap/plugin/servicenow/restapi/RestAPIClient.java]: No, we are not using It !! Deleted now.

**Pull request merged**